src/components/Main.js

```diff
@@ -17,72 +21,179 @@ import Text from './Shared/UniText';
 import UserList from './UserList/UserList';
 import UserUtil from '../utils/UserUtil';
 
+const mainPageTitle = {
+  DEFAULT: -1,
+  HOME: 0,
+  TOURNAMENT: 1,
+  MYCONNECTION: 2,
+  CHAT: 3,
+  MYPROFILE: 4,
+};
+
+const activityPageTitle = {
+  DEFAULT: -1,
+  NEWREQUESTS: 0,
+  CONNECTED: 1,
+};
+
 class Main extends Component {
   constructor(props) {
     super(props);
+
+    this.state = {
+      currentMainPage: mainPageTitle.DEFAULT,
+      currentActivityPage: activityPageTitle.DEFAULT,
+    };
+
+    this.isConnected = false;
   }
 
   componentDidMount() {
-    this.initSendBird();
+    AppState.addEventListener('change', this.onAppStateChange.bind(this));
+    this.initSendBird((user, error)=> {
+
+      if (user.nickname !== this.props.me.name ||
+        user.profileUrl !== this.props.me.profile_picture) {
+
+        //Sendbird has an issue in updateCurrentUserInfo() API right after connect() API.
+        setTimeout(() => {
+          this.sb.updateCurrentUserInfo(this.props.me.name, this.props.me.profile_picture);
+        }, 1000);
+      }
+
+      NetInfo.isConnected.addEventListener('change', this.onConnectionStateChange.bind(this));
+
+      this.notificationUnsubscribe = Fcm.on('notification', this.onNotificationReceived.bind(this));
+      Fcm.getInitialNotification().then(notif => {
```

Comment:
If you use '{', '}', it is good to define a function like this.
() => {}
Use '(', ')'
this is airbnb style guide.